client/src/pages/Login.js

```diff
@@ -1,67 +1,210 @@
 import axios from "axios";
 import React from "react";
 import { useHistory } from "react-router-dom";
-
+import MuiAlert from "@material-ui/lab/Alert";
 import { SET_USER } from "../contexts/types";
 import { UserContext } from "../contexts/UserContext";
+import {
+  Grid,
+  Paper,
+  TextField,
+  Link,
+  Button,
+  Snackbar,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Logo from "../components/Logo";
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+const useStyles = makeStyles((theme) => ({
+  snackbar: {
+    width: "100%",
+    "& > * + *": {
+      marginTop: theme.spacing(2),
+    },
+  },
+  root: {
+    height: "100vh",
+  },
+  image: {
+    backgroundImage:
+      "url(https://i.ibb.co/YRBjvKT/f1d2e32ad77c9c983af281c12eee46567109a4f6.png)",
```

Comment:
instead of uploading this image to a 3rd party, put it under `/public` folder in `client`, this should be accessible in your client app